vuepress/docs/next/tutorials/consume/multitenancy-tutorial.md

```diff
@@ -0,0 +1,239 @@
+``` yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    entando.org/quickstart-appbuilder-path: /app-builder/
+    entando.org/quickstart-de-path: /digital-exchange
+    entando.org/quickstart-pn-3c07adf0-fac54a9f-entando3323-pn-3c07adf0-fa-path: /entando-epc-bootstrap-3c07adf0/app-builder-menu-bff
+    entando.org/quickstart-server-path: /entando-de-app
+  generation: 4
+  labels:
+    EntandoApp: quickstart
+  name: YOUR-TENANT1-ID-quickstart-ingress
+  namespace: entando
+spec:
+  rules:
+  - host: YOUR-TENANT1-ID.YOUR-HOSTNAME
+    http:
+      paths:
+      - backend:
+          service:
+            name: quickstart-ab-service
+            port:
+              number: 8081
+        path: /app-builder/
+        pathType: Prefix
+      - backend:
+          service:
+            name: quickstart-service
+            port:
+              number: 8080
+        path: /entando-de-app
+        pathType: Prefix
+      - backend:
+          service:
+            name: quickstart-cm-service
+            port:
+              number: 8083
+        path: /digital-exchange
+        pathType: Prefix
+      - backend:
+          service:
+            name: pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-service
+            port:
+              number: 8081
+        path: /entando-epc-bootstrap-3c07adf0/app-builder-menu-bff
+        pathType: Prefix
+```
+
+### Create the Tenant ConfigMap 
+
+Here is an example: 
+``` yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: YOUR-TENANT1-CONFIGMAP
+data:
+  ENTANDO_TENANTS: >-
+        [
+            {
+                "tenantCode": "YOUR-TENANT1-ID", # default subdomain name 
+                "fqdns": "YOUR-TENANT1-ID.YOUR-HOSTNAME" # value string and comma separated domains
+                "kcEnabled": true,
+                "kcAuthUrl": "https://YOUR-HOSTNAME/auth",
+                "kcRealm": "YOUR-TENANT1-REALM",
+                "kcClientId": "quickstart",
+                "kcClientSecret": "YOUR-TENANT1-KC-SECRET",
+                "kcPublicClientId": "entando-web",
+                "kcSecureUris": "",
+                "kcDefaultAuthorizations": "",
+                "dbDriverClassName": "org.postgresql.Driver",
+                "dbUrl": "YOUR-POSTGRESQL-TENANT1-URL",
+                "dbUsername": "postgres",
+                "dbPassword": "e7d60efa865c4510",
+                "cdsPublicUrl": "https://YOUR-FQDN/YOUR-TENANT1-CODE ", # will depend on your DNS entry
+                "cdsPrivateUrl": "https://YOUR-TENANT1-CDS-SERVICE.YOUR-NAMESPACE.svc.cluster.local:8080/",
+                "cdsPath": "api/v1",
+                "solrAddress": "https://YOUR-SOLR-SERVICE.YOUR-NAMESPACE.svc.cluster.local:YOUR-SOLR-PORT/solr",
+                "solrCore": "TENANT1-CORE-NAME"
+            },
+            {
+                "tenantCode": "YOUR-TENANT2-CODE",
```

Comment:
is it intended to have a config for the tenant-1 that contains more that the tenant-1 values?